homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -220,6 +221,9 @@ def device_state_attributes(self):
                     / 3600),
                 ATTR_FILTER_LEFT: int(
                     self.consumable_state.filter_left.total_seconds()
+                    / 3600),
+                ATTR_SENSOR_DIRTY: int(
+                    self.consumable_state.sensor_dirty.total_seconds()
```

Comment:
The patch looks fine to me, however, I'm wondering whether it would make more sense to expose "left" type of value similar to other sensors? This would require 1) finding out what's the total use time for the sensor before it needs to be cleaned and 2) implementing it in python-miio.